modules/installation-azure-config-yaml.adoc

```diff
@@ -103,9 +107,15 @@ ifdef::private,gov[]
 endif::private,gov[]
 ifndef::gov[]
     cloudName: AzurePublicCloud
+      osDisk:
+        diskSizeGB: 512
+        diskType: premium_LRS
 endif::gov[]
 ifdef::gov[]
     cloudName: AzureUSGovernmentCloud <14>
+      osDisk:
```

Comment:
osDisk should be has same indent with cloudName

Reply:
Thank you! Should be all fixed now